--- a/usr/share/voicecall-ui-jolla/main.qml
+++ b/usr/share/voicecall-ui-jolla/main.qml
@@ -288,6 +288,8 @@ ApplicationWindow {
             iface: "com.jolla.voicecall.ui"
             function show(args) {
                 ensureCallLogPage(function() { main.activate() })
+                dialer.active = true
+                numberFieldActive = true
             }
             function showOngoing() {
                 if (main.state == 'null' || main.state == 'disconnected') {
